pyproject.toml

```diff
@@ -44,6 +44,7 @@ dependencies = [
     "termcolor==2.3.0",
     "psutil==5.9.5",
     "protobuf==3.20.0",
+    "nest-asyncio",
```

Comment:
We do not need to add a version for this. 🤔

Reply:
See https://github.com/neulab/prompt2model/issues/296